goentri.com.fairmint.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
This file is in #306, so should not be here, right?

Reply:
Yes, you are right, it's deleted. Thank you.